src/components/users/user-signin-social/ko/runtime/signin-aad-b2c.ts

```diff
@@ -54,8 +54,12 @@ export class SignInAadB2C {
         try {
             await this.aadService.signInWithAadB2C(this.clientId(), this.authority(), this.instance(), this.signInPolicy());
             await this.router.navigateTo(Constants.homeUrl);
+            const event = new CustomEvent("validationerror", {detail: {msgs: [], from: "socialAcc"}});
+            document.dispatchEvent(event);
```

Comment:
it should be this.eventManager.dispatchEvent("onValidationErrors", { ... }) instead of document.dispatchEvent(...). Here eventManager is IEventManager.